xtext-android-gradle-plugin/src/main/java/org/xtext/gradle/android/XtextAndroidBuilderPlugin.xtend

```diff
@@ -78,7 +78,9 @@ class XtextAndroidBuilderPlugin implements Plugin<Project> {
 			generatorTask.dependsOn(variant.outputs.map[processResources])
 			variant.javaCompiler.doLast(new Action<Task>() {
 				override void execute(Task it) {
-					generatorTask.installDebugInfo()
+					if (it instanceof AbstractCompile) {
+						generatorTask.installDebugInfo(destinationDir)
```

Comment:
@oehme i am not sure if i need to do the project.files as i have removed it from below here

Reply:
but i dont think so.

Reply:
or can destinationDir be relative. in my experiments it was absolute

Reply:
It'll be absolute, so no worries here :)